reference/price_lists.v3.yml

```diff
@@ -2,179 +2,86 @@ swagger: '2.0'
-
+    Additionally, [Price List Assignments](https://developer.bigcommerce.com/api-reference/store-management/price-lists/price-lists-records/) can be created to assign Price Lists to a specific [Channel](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api). Price lists assigned to a channel apply to all shoppers on that channel, unless there is a more specific assignment.
 
-    Additionally, [Price List
-    Assignments](https://developer.bigcommerce.com/api-reference/store-management/price-lists/price-lists-records/)
-    can be created to assign Price Lists to a specific
-    [Channel](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api).
-    Price lists assigned to a channel apply to all shoppers on that channel,
-    unless there is a more specific assignment.
+    If an active Price List does not contain prices for a variant then the Catalog pricing will be used. The association of a Price List to a Customer Group can be done either via the Control Panel or using the [Customer Groups API.](/api-reference/customer-subscribers/customers-api)
 
+    Price Lists will provide overridden price values to the Stencil storefront. Final price display can be further customized within the Stencil template. See the [Price Object](https://stencil.bigcommerce.com/docs/price-object-properties) in Stencil for further documentation.
 
-    If an active Price List does not contain prices for a variant then the
-    Catalog pricing will be used. The association of a Price List to a Customer
-    Group can be done either via the Control Panel or using the [Customer Groups
-    API.](/api-reference/customer-subscribers/customers-api)
-
-
-    Price Lists will provide overridden price values to the Stencil storefront.
-    Final price display can be further customized within the Stencil template.
-    See the [Price
-    Object](https://stencil.bigcommerce.com/docs/price-object-properties) in
-    Stencil for further documentation.
-
-
-    To learn more about Price Lists, see
-    [here](/api-docs/price-lists/price-list-overview).
-
+    To learn more about Price Lists, see [here](/api-docs/price-lists/price-list-overview).
 
     ## Price list assignments
 
-
     ### Order of operations
 
-
     The `Price List Assignment` Pricing Order of Operations is as follows:
 
-
-    **IF** `Price list` assigned to current `Customer Group` **AND** `Price
-    List` assigned to current `Channel`:
-
-    * Use this Price List -- any prices not found fall back to the catalog price
-    (or in the case of multi-currency, auto-converted prices)
-
+    **IF** `Price list` assigned to current `Customer Group` **AND** `Price List` assigned to current `Channel`:
+    * Use this Price List -- any prices not found fall back to the catalog price (or in the case of multi-currency, auto-converted prices)
 
     **ELSE IF**: `Price List` assigned to current `Channel`:
```

Comment:
I think this needs to be removed- the order is 
cg + channel
cg
cg discounts
channel price list